website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -213,36 +200,84 @@ This property contains all the details of the edited rows. It has the following
 | ------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| **onSubmit**  | Action that gets triggered when the user moves away from editing a cell by pressing the enter key or clicking outside. |
-| **onSave**    | Action that gets triggered when the user clicks the save button for a cell or a new row.                                                       |
-| **onDiscard** | Action that gets triggered when user clicks discard button for a cell or a new row.                                                       |
+| **onSubmit**  | Action that gets triggered when the user moves away from editing a cell by pressing the enter key or clicking outside of the cell. |
+| **onSave**    | Action that gets triggered when the user clicks the save button for a cell or a new row.                               |
+| **onDiscard** | Action that gets triggered when user clicks discard button for a cell or a new row.                                    |
+
+### Enable editing cells
+
+The column list on the Table property pane has a checkbox to make that column editable.
+
+Click the checkbox inside a column card makes all the cells in that column editable. The editable checkbox at the top of the list can be checked to make all the editable columns editable.
+
+Alternatively, A column can be editable by turning on `Editable` property inside the column's settings.
+
+![](</img/Editable_preoperty.png>)
+
+You can also use JS to control the `Editable` property. For example, you might use this feature to make only a subset of the cells editable:
+
+![](</img/Using_JS_in_Editable.png>)
+
+```javascript
+// in the Editable property
+{{ currentRow.status === "pending" }} // only "pending rows are editable
+```
+
+Once a column has been made editable, a pencil icon appears on the column header as an indicator.
+
+![](</img/Edit_icon_on_column_header.png>)
+
+### How to edit a cell
+
+Currently, **four column types** support inline editing: these  are **Text, Number, Switch, and Checkbox**. When one of these types of cells is made editable, it displays a pencil edit icon when the user hovers their cursor over it. Click this icon or double-click the cell to begin making changes.
+
+![](</img/Screen_Recording_2022-09-30_at_12_21_13_PM_AdobeExpress.gif>)
+
+Once the user is finished editing, they can move away from the edit mode in two ways:
+
+1. Pressing the **Enter** key or **clicking outside the cell** keeps the new value and closes the input box.
+2. Pressing the **Escape** key discards the new value and closes the input box.
+
+:::info
+To save the edits made to your table rows, you'll need to set the table's [**Update mode**](#properties) to [**Single row**](#update-mode-single-row) or [**Multi row**](#update-mode-multi-row) mode and then configure your Save buttons to execute queries.
+:::
+
+### Perform an action when a cell is edited
```

Comment:
sub-heading not required. it is continuous flow of steps that the user has to perform ot update the row after cell contents are edited